### DISCUSSION TITLE
侧边栏社交图标问题

### DISCUSSION THREAD

**Discussion by ctdrm** [MaLuns/hexo-theme-async#45] [Q&A 疑难杂症] [answered]
![image](https://user-images.githubusercontent.com/100547590/209314161-daa86d86-8dc4-4b89-8918-5af06ae40249.png)
在根目录下配置文件中，只配置一个邮箱，但是显示页面中除了邮箱，后面还会多出来一个小蓝鸟

**Comment by MaLuns** [comment, marked as answer]
默认配置里两个图标，只设置一个情况下， Hexo 合并配置只覆盖了第一个。

已在 v1.2.9 修复
